compiler/test/suites/imports.re

```diff
@@ -169,16 +170,20 @@ describe("imports", ({test}) => {
     "import_same_module_unify2",
     "import *, TList from \"tlists\"; Cons(5, TList.Empty)",
   );
-  /* import filepath tests */
-  assertSnapshot("import_relative_path", "import * from \"./exportStar\"; x");
+  /* import filepath tests (test names are important to make relpaths work) */
```

Comment:
We should probably make two new files for these tests and use `assertFileshot`, since it's more realistic anyway 🙂